apps/pip/account-management/image-policy.yaml

```diff
@@ -2,14 +2,6 @@ apiVersion: image.toolkit.fluxcd.io/v1alpha2
 kind: ImagePolicy
 metadata:
   name: pip-account-management
-  annotations:
-    hmcts.github.com/prod-automated: disabled
 spec:
-  filterTags:
-    extract: $ts
-    pattern: '^prod-[a-f0-9]+-(?P<ts>[0-9]+-[0-9]+)'
   imageRepositoryRef:
-    name: pip-account-management
-  policy:
-    alphabetical:
-      order: asc
\ No newline at end of file
+    name: pip-account-management
```

Comment:
is name meant to be in this file twice?

Reply:
same for the other files in this pr

Reply:
yeah one is the name of the `imagePolicy` resource and the other is the reference to the `imageRepository`, they just happen to be the same :)